src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.TransportNames.approved.txt

```diff
@@ -6,14 +6,18 @@ namespace ServiceControlInstaller.Engine.Instances
     {
         public const string AmazonSQS = "AmazonSQS";
         public const string AzureServiceBus = "Azure Service Bus";
-        public const string AzureServiceBusEndpointOrientedTopology = "Azure Service Bus - Endpoint-oriented topology (Legacy)";
+        public const string AzureServiceBusEndpointOrientedTopologyDeprecated = "DEPRECATED: Azure Service Bus - Endpoint-oriented topology (Legacy)";
+        public const string AzureServiceBusEndpointOrientedTopologyLegacy = "Azure Service Bus - Endpoint-oriented topology (Legacy)";
         public const string AzureServiceBusEndpointOrientedTopologyOld = "Azure Service Bus - Endpoint-oriented topology (Old)";
-        public const string AzureServiceBusForwardingTopology = "Azure Service Bus - Forwarding topology (Legacy)";
+        public const string AzureServiceBusForwardingTopologyDeprecated = "DEPRECATED: Azure Service Bus - Forwarding topology (Legacy)";
```

Comment:
I would concatenate the prefix constant here 🙂

Reply:
That is the approval file :D I can't do anything here